bin/fa5-upgrade.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/sh
+
+echo "Please enter your FontAwesome5 npm token:"
+
+read fa5_token
+
+echo "Setting up npm config"
+
+npm config set "@fortawesome:registry" https://npm.fontawesome.com/
+npm config set "//npm.fontawesome.com/:_authToken" $fa5_token
+
+echo "Creating temporary folder"
+
+TEMP_DIR=`mktemp -d -t rnvi`
+echo "Created folder $TEMP_DIR"
+pushd $TEMP_DIR
+
+echo "Downloading FontAwesome5 Pro"
+
+npm pack @fortawesome/fontawesome-pro
+tar -xzf fortawesome-fontawesome-pro*
```

Comment:
AFAIK `npm pack` will output the file name, so you can use that instead of globing.